ubuntudesign/www.ubuntu.com
# Cu download cloud

## Done

Updated the content of  `/download/cloud`

## QA

Check `/download/cloud` in browser and check that the [copy](https://docs.google.com/document/d/1y3uaLNn7dxZKcLq2OmVJ96o3NOqn4OTJflty2GmxV9I/edit#) is correct.
